includes/activitypub.js

```diff
@@ -0,0 +1,52 @@
+(function($) {    
```

Comment:
I would not mix up PHP and JS code. Maybe we should add an assets folder to the root, including JS/CSS/... files:

`/assets/js/activitypub.js`